images/k8ssandra-system-logger/README.md

```diff
@@ -0,0 +1,28 @@
+<!--monopod:start-->
+# k8ssandra-system-logger
+| | |
+| - | - |
+| **OCI Reference** | `cgr.dev/chainguard/k8ssandra-system-logger` |
+
+
+* [View Image in Chainguard Academy](https://edu.chainguard.dev/chainguard/chainguard-images/reference/k8ssandra-system-logger/overview/)
+* [View Image Catalog](https://console.enforce.dev/images/catalog) for a full list of available tags.
+* [Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
+
+---
+<!--monopod:end-->
+
+<!--overview:start-->
+Sidecar for DataStax Kubernetes Operator for Apache Cassandra
+<!--overview:end-->
+
+<!--getting:start-->
+## Download this Image
+The image is available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/k8ssandra-system-logger:latest
+```
+<!--getting:end-->
+
+<!--body:start--><!--body:end-->
```

Comment:
Can we add some more docs here on how to use the image? i.e perhaps with cass-operator helm chat which seems to be the most common? The chart does support multiple images (a lot of we also have), but the example could just show how to override just this single image with the chart.

If we think a docker example would be good too then we can also include

Reply:
Thanks, done!